### PR TITLE
Updates010617

### PR DESCRIPTION
**Pull request by RDaviesDG**
Completed updates required to documentation on 1/9/2017.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -165,17 +171,29 @@
...
             <h3>Support or Contact</h3>
 
             <p>Having issues with DeepCore? Email us at <a href="mailto:deepcore@digitalglobe.com">
                 <strong>DeepCore</strong> </a> or
                 visit our <a href="https://github.com/DigitalGlobe/DeepCore/issues"><strong>Issues</strong></a> page to
                 log a bug or get some help. Also, check out the DeepCore documentation <a
                         href="website/documentation/html/index.html"> <strong>here</strong>.</a></p>
+            <p class="dg-copyright"><a href="http://www.digitalglobe.com">©2017 DigitalGlobe</a></p>
         </section>
+        <footer><</footer>
```
Comment (geovations) [inline index.html]: Is that extra '<' meant to be there?

File: index.html [hunk 2]
```diff
@@ -201,8 +219,11 @@
             <p class="repo-owner"><a href="https://github.com/DigitalGlobe/DeepCore">DeepCore</a> is maintained by <a
                     href="https://github.com/DigitalGlobe">DigitalGlobe</a>.</p>
         </aside>
+
     </div>
 
+
+
```
Comment (geovations) [inline index.html]: Why the extra whitespace?

File: index.html [hunk 3]
```diff
@@ -94,6 +98,8 @@
                 <a id="creating-pages-manually" class="anchor" href="#creating-pages-manually" aria-hidden="true"><span
                         aria-hidden="true" class="octicon octicon-link"></span></a>Sample Application - OpenSpaceNet
             </h3>
+            <img src="website/images/airliner_detection_example2.png"/>
+            <p class="image-caption">Object detection being used to find aircraft on the ground at Hartsfield-Jackson airport in Atlanta, GA</p>
```
Comment (KevinMcGee) [inline index.html]: This is not an image of the Hartsfield-Jackson airport, please remove that

**Comment by RDaviesDG** [comment]
Updated following the requested changes.

**Comment by RDaviesDG** [comment]
Done.

File: index.html [hunk 4]
```diff
@@ -27,12 +28,13 @@
 <div id="content-wrapper">
     <div class="inner clearfix">
         <section id="main-content">
-            <h3>
+            <h2>
                 <a id="welcome-to-github-pages" class="anchor" href="#welcome-to-github-pages" aria-hidden="true"><span
-                        aria-hidden="true" class="octicon octicon-link"></span></a>Welcome to DeepCore</h3>
+                        aria-hidden="true" class="octicon octicon-link"></span></a>Welcome to DeepCore</h2>
+            <img src="website/images/atlanta_airport_detection_example.png"/>
+            <p class="image-caption">Object detection being used to find aircraft on the ground. </p>
```
Comment (KevinMcGee) [inline index.html]: You can keep Atlanta, GA here. This is the only example I sent you that is in Atlanta

**Pull request merged**